js/interface/map-panel.js

```diff
@@ -25,12 +25,12 @@ Fliplet.InteractiveMap.component('map-panel', {
   data() {
     return {
       updateDebounced: _.debounce(this.updateDataSource, 1000),
-      appId: _.keys(__widgetData)[0],
-      dataSourceId: undefined,
+      appId: Fliplet.Widget.getDefaultId(),
```

Comment:
the widget Id is not the app Id

Reply:
But it does work for me.

Reply:
In what sense it does work? Does `Fliplet.Widget.getDefaultId()` return the same ID as `Fliplet.Env.get('appId')`?

Reply:
I use it just to refresh build after changes to data source are done, should I rename it to widgetId?

Reply:
@MaksymShokin I see, yes you should rename the variable to `widgetInstanceId`

Reply:
@squallstar Fixed